src/handlers/store-transactions/handler.ts

```diff
@@ -18,7 +18,6 @@ export const handler = async (event: SQSEvent): Promise<Response> => {
 };
 
 async function storeRecord(record: SQSRecord): Promise<void> {
-  console.log("storing record " + JSON.stringify(record));
```

Comment:
one of the int test checks event id in the logs. For debugging purpose can we log eventid ?

Reply:
I've added it back it but moved it down to just before the event is actually stored. I thought it could be misleading where it was before as we would potentially log that we were storing something but then throw an error before we even tried to store it.